server/rooms.ts

```diff
@@ -824,6 +829,29 @@ export abstract class BasicRoom {
 			}
 		}
 	}
+	sanitizeSection(section: string) {
+		const target = toID(section);
+		if (!RoomSections.sections.includes(target as any)) {
+			throw new Chat.ErrorMessage(`"${target}" is not a valid room section. Valid categories include: ${RoomSections.sections.join(', ')}`);
+		}
+		return target as RoomSection;
+	}
+	adjustSection(newSection: string) {
```

Comment:
Is there a reason this isn't `setSection`?

Reply:
not really